README.md

```diff
@@ -1 +1,39 @@
-# form-project-mint
\ No newline at end of file
+# Form Project Mint
+
+> The project is about a Mint form for users.
+
+![screenshot](IMG/mint_screenshot.JPG)
+
+The project has a full description form with a title and a field for email, phone number and a password
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://deikdesign.github.io/form-project-mint/)
+
+Prerequisites - Chrome browser
+*Setup
+ - $ cd "folder location"
+ - $ git clone https://github.com/deikdesign/form-project-mint.git
+* Install
+* Usage 
+* Run tests - HTML validator, w3 validation
+* Deployment 
```

Comment:
- [ ]  [Optional] please either update this section with the needed information or delete it